cmake/modules/FindD3D12_WSL.cmake

```diff
@@ -0,0 +1,29 @@
+ # Check if in WSL and if has DirectX driver and runtime
+if (EXISTS "/usr/lib/wsl/lib/")
```

Comment:
Not indenting the if block is a bit confusing, but to avoid indenting the whole file you can invert the condition and have the block just call return() to stop processing the file.